### PR TITLE
Add support for CSVs in SVObs format.

### PR DESCRIPTION
**Pull request by keyurva**

### CODE REVIEW

File: simple/stats/config.py [hunk 1]
```diff
@@ -83,6 +86,17 @@ def import_type(self, input_file_name: str) -> ImportType:
...
+  def format(self, input_file_name: str) -> ImportType | None:
+    format_str = self._input_file(input_file_name).get(_FORMAT_FIELD)
+    if not format_str:
+      return None
+    if format_str not in iter(InputFileFormat):
+      raise ValueError(f"Unsupported format: {format_str} ({input_file_name})")
+    return InputFileFormat(format_str)
+
+  def column_mappings(self, input_file_name: str) -> dict[str, str]:
+    return self._input_file(input_file_name).get(_COLUMN_MAPPINGS_FIELD, {})
```
Comment (shifucun) [inline simple/stats/config.py:98]: should self._input_file(input_file_name) be computed once and stored as a map in "__init__.py"? This is repeatedly doing a few times in each function.

File: simple/stats/variable_per_row_importer.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+_COLUMNS = [
+    constants.COLUMN_ENTITY, constants.COLUMN_VARIABLE, constants.COLUMN_DATE,
+    constants.COLUMN_VALUE
+]
+_DEFAULT_COLUMN_MAPPINGS = {x: x for x in _COLUMNS}
+
+
+class VariablePerRowImporter(Importer):
+  """Imports a single observations input file where variables are specified in rows (aka "SVObs").
```
Comment (shifucun) [inline simple/stats/variable_per_row_importer.py:35]: The class name does not seem to match the description? This is also about observation but it contrasts with "ObservationsImporter"?

File: simple/tests/stats/test_data/runner/config/config_driven.json [hunk 3]
```diff
@@ -10,6 +10,12 @@
       "importType": "observations",
       "entityType": "Country",
       "provenance": "Provenance1 Name"
+    },
+    "svobs.csv": {
```
Comment (shifucun) [inline simple/tests/stats/test_data/runner/config/config_driven.json:14]: not using "svobs"? This is more of an internal acronym, maybe something matching the file format config.

File: simple/tests/stats/test_data/runner/config/config_driven.json [hunk 4]
```diff
@@ -10,6 +10,12 @@
       "importType": "observations",
       "entityType": "Country",
```
Comment (shifucun) [inline simple/tests/stats/test_data/runner/config/config_driven.json:11]: should this one also have a "format" field?

File: simple/stats/config.py (hunk 1 above)
Comment (keyurva) [reply simple/stats/config.py:98]: Good observation. It is computed once and cached: https://github.com/datacommonsorg/import/blob/e8b0ada6e9a494698c9d26fd9ebafcba350e3f5b/simple/stats/config.py#L57-L61

File: simple/stats/variable_per_row_importer.py (hunk 2 above)
Comment (keyurva) [reply simple/stats/variable_per_row_importer.py:35]: True but `VariablePerRowObservationsImporter` felt too long so went with `VariablePerRowImporter`.

File: simple/tests/stats/test_data/runner/config/config_driven.json (hunk 3 above)
Comment (keyurva) [reply simple/tests/stats/test_data/runner/config/config_driven.json]: Done.

File: simple/tests/stats/test_data/runner/config/config_driven.json (hunk 4 above)
Comment (keyurva) [reply simple/tests/stats/test_data/runner/config/config_driven.json:11]: No. This has the default variable per column format.

Comment (shifucun) [reply simple/tests/stats/test_data/runner/config/config_driven.json:11]: I feel it's more clear to always set the format.

**Review by shifucun — APPROVED** [review]

**Review by pradh — COMMENTED** [review]
(Please go with Bo's approval, but just need to make sure this covers all SVObs props)

File: simple/tests/stats/test_data/runner/config/config_driven.json (hunk 4 above)
Comment (keyurva) [reply simple/tests/stats/test_data/runner/config/config_driven.json:13]: SG - added.

File: simple/tests/stats/test_data/runner/input/input_dir_driven/variable_per_row.csv [hunk 5]
```diff
@@ -0,0 +1,5 @@
+entity,variable,date,value
```
[the remark this answers is not in the document]
Comment (pradh) [reply simple/tests/stats/test_data/runner/input/input_dir_driven/variable_per_row.csv:1]: hmm, not sure I fully follow -- that's referring to properties of StatisticalVariable (there are more than those too), and I was referring to properties of StatVarObservation?

[the remark this answers is not in the document]
Comment (keyurva) [reply simple/tests/stats/test_data/runner/input/input_dir_driven/variable_per_row.csv:1]: Ah. Can you point me to the SVObs properties you're talking about and I'll incorporate them?

[the remark this answers is not in the document]
Comment (pradh) [reply simple/tests/stats/test_data/runner/input/input_dir_driven/variable_per_row.csv:1]: StatVarObservation table: https://github.com/datacommonsorg/mixer/blob/eb548b3f453d65b3d81a06b09114e74b9ce89ce4/deploy/mapping/base.mcf#L47-L56

Can you send me a pointer to the list of StatVar properties supported in RSI?

[the remark this answers is not in the document]
Comment (keyurva) [reply simple/tests/stats/test_data/runner/input/input_dir_driven/variable_per_row.csv:1]: Actually, the [properties field](https://github.com/datacommonsorg/import/blob/e8b0ada6e9a494698c9d26fd9ebafcba350e3f5b/simple/stats/config.md#properties) is a generic dict so any property can be specified. Any specific you're looking for here?

[the remark this answers is not in the document]
Comment (pradh) [reply simple/tests/stats/test_data/runner/input/input_dir_driven/variable_per_row.csv:1]: I see, ok, where does the validation for having expected properties happen?

Is that the same for SVObs format too?  (i.e., no restriction on the property)

[the remark this answers is not in the document]
Comment (keyurva) [reply simple/tests/stats/test_data/runner/input/input_dir_driven/variable_per_row.csv:1]: Yes no restriction. Currently there is no validation.

Which properties should be validated for SVObs? I can add them when I wire in writing entities for them.

**Pull request merged**